django/thunderstore/usermedia/models/usermedia.py

```diff
@@ -0,0 +1,90 @@
+import copy
+
+import ulid2
+from django.conf import settings
+from django.db import models
+from django.db.models import Q
+from django.utils import timezone
+
+from thunderstore.core.mixins import TimestampMixin
+from thunderstore.core.types import UserType
+from thunderstore.core.utils import ChoiceEnum
+
+
+class UserMediaQueryset(models.QuerySet):
+    def expired(self):
+        return self.exclude(Q(expiry=None) | Q(expiry__gt=timezone.now()))
+
+    def active(self):
+        return self.exclude(Q(expiry__lte=timezone.now()))
+
+
+class UserMediaStatus(ChoiceEnum):
+    initial = "initial"
+    upload_created = "upload_initiated"
+    upload_error = "upload_error"
+    upload_complete = "upload_complete"
+    upload_aborted = "upload_aborted"
```

Comment:
Should these keys be UPPER_CASE?

Reply:
that's a common convention yes, but no if we want to keep it consistent with the rest of the codebase